src/libraries/Common/tests/System/DirectoryServices/LdapConfiguration.cs

```diff
@@ -110,7 +118,14 @@ internal static LdapConfiguration GetConfiguration(string configFile)
 
                     child = config.Element("Password");
                     if (child != null)
-                        password = child.Value;
+                    {
+                        string val = child.Value;
+                        if (val.StartsWith("%") && val.EndsWith("%"))
```

Comment:
I believe you need to check for the string length here be greater than 1.

Reply:
"".StartsWith("%") just returns false.